dashboard/word_cloud_dashboard.py

```diff
@@ -0,0 +1,101 @@
+"""A file to generate word clouds based on article word frequency by source."""
+
+import re
+
+import streamlit as st
+from wordcloud import WordCloud
+import matplotlib.pyplot as plt
+from dotenv import load_dotenv
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+import nltk
```

Comment:
Since you are being specific about nltk imports on line 9 and 10 maybe also do so here?
e.g: `from nltk import download as nltk_download`

Reply:
Good idea, I'll do that now!